### PR TITLE
Switch to Terser for JS minification

### PR DESCRIPTION
**Pull request by willdurand**
Fixes https://github.com/mozilla/addons-server/issues/13682

### CODE REVIEW

File: src/olympia/lib/settings_base.py [hunk 1]
```diff
@@ -85,10 +85,11 @@ def path(*folders):
 CLEANCSS_BIN = env(
     'CLEANCSS_BIN', default='node_modules/less/bin/lessc')
 
-# Path to uglifyjs (our JS minifier).
+# Path to our JS minifier.
 # Set as None to use YUI instead (at your risk).
```
Comment (bobsilverberg) [inline src/olympia/lib/settings_base.py:89]: Is this comment still valid? I didn't see anything in `_minify` that suggests this, nor anywhere else that `JS_MINIFIER_BIN` is used.

Comment (willdurand) [reply src/olympia/lib/settings_base.py:89]: Looks like this was indeed removed in https://github.com/mozilla/addons-server/pull/8694

File: static/js/impala/stats/stats.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
   'use strict';
 
   // `$` is passed by jQuery itself when calling `jQuery(stats_stats)`.
-  var stats_stats = function ($, injectedSessionStorage) {
+  const stats_stats = function ($, injectedSessionStorage) {
```
Comment (eviljeff) [inline static/js/impala/stats/stats.js:5]: 😱 `const`!

Comment (bobsilverberg) [reply static/js/impala/stats/stats.js:5]: When's the PR coming to convert everything to `const`? 😉

Comment (willdurand) [reply static/js/impala/stats/stats.js:5]: https://github.com/mozilla/addons-server/pull/15131

**Pull request merged**